DetourCrowd/Source/DetourCrowd.cpp

```diff
@@ -386,7 +386,7 @@ bool dtCrowd::init(const int maxAgents, const float maxAgentRadius, dtNavMesh* n
 	m_maxAgents = maxAgents;
 	m_maxAgentRadius = maxAgentRadius;
 
-	dtVset(m_ext, m_maxAgentRadius*2.0f,m_maxAgentRadius*1.5f,m_maxAgentRadius*2.0f);
+	dtVset(m_halfExtents, m_maxAgentRadius*2.0f,m_maxAgentRadius*1.5f,m_maxAgentRadius*2.0f);
```

Comment:
I see, that makes sense. However it is a little nondeterministic since `findNearestPoly` can return polys that do not actually touch the extents (on account of `queryPolygons` using AABB's for the polys internally).

@aymarfisherman do you mind making the renaming suggested, now that you're renaming here already?